contact.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Contact</title>
+    <link rel="stylesheet" href="homestyle.css">
+	<meta charset="utf-8">
```

Comment:
This meta tag appears twice! Also don't forget to add in the responsive meta tag as per my other comment